### PR TITLE
Links from dept homepages to related ministers

### PR DESCRIPTION
**Pull request by micapam**
Links from department homepages to related ministers. Administrable via admin.

Still to do is to include the name of the minister next to the link (as per prototype).

### CODE REVIEW

File: app/views/templates/_related_ministers.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- if related_ministers.present?
```
Comment (ltankey) [inline app/views/templates/_related_ministers.html.haml]: Does passing in the local variable `object` automatically set a variable to the name of the template?

Comment (micapam) [inline app/views/templates/_related_ministers.html.haml]: Yes.

Comment (tessereth) [inline app/views/templates/_related_ministers.html.haml]: yes

**Comment by ltankey** [comment]
LGTM

File: app/views/templates/_related_ministers.html.haml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  %h2 Our ministers
+  %ul.list-horizontal
+    - related_ministers.each do |minister|
+      %li
+        %figure
+          %img{src: minister.image_url}
+        %article
+          %h3
+            =link_to(minister.name, nodes_path(path: minister.home_node.path))
```
Comment (ltankey) [inline app/views/templates/_related_ministers.html.haml]: Should this be `public_node_path`?

Comment (micapam) [inline app/views/templates/_related_ministers.html.haml]: Indeed!

**Comment by tessereth** [comment]
LGTM

**Pull request merged**